Dockerfile

```diff
@@ -11,13 +11,13 @@ ENV PATH=/home/node/.npm-global/bin:$PATH
 ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
 
 USER node
-RUN npm install -g scuttlebot-release@13.0.0-2
+RUN npm install -g ssb-server@14.1.2
 
 EXPOSE 8008
 
 HEALTHCHECK --interval=30s --timeout=30s --start-period=10s --retries=10 \
   CMD sbot whoami || exit 1
```

Comment:
`ssb-server whoami`

Reply:
`ssb-server` still exports the `sbot` command, this is me being lazy but it should work.

Reply:
Isn't using sbot deprecated? IIRC it throws a warning when used. Might as well change it to set a good example for the community.